src/test/java/Homework17_PB.java

```diff
@@ -1,5 +1,6 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 public class Homework17_PB extends BaseTest{
```

Comment:
Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`class HomeWork17`